components/dashboard/src/menu/Menu.tsx

```diff
@@ -98,31 +106,38 @@ export default function Menu() {
                                         />
                                     </li>
                                 )}
-                                {isGitpodIo && (
-                                    <li className="cursor-pointer">
-                                        <PillMenuItem name="Feedback" onClick={handleFeedbackFormClick} />
+                                {!isDataOps && (
+                                    <li>
+                                        <a
+                                            href="/"
+                                            className="flex items-center gap-x-1 text-sm text-gray-500 dark:text-gray-400 hover:text-gray-900 dark:hover:text-white"
+                                        >
+                                            <ProductLogo className="h-4 w-auto" />
+                                            <span>Gitpod Classic</span>
+                                        </a>
                                     </li>
```

Comment:
I'd say let's get rid of the link here for now, since having it link to something feels like it should link to some product marketing page. We already have the "Workspaces" link, which should serve the purpose of going "Home", that users should be familiar with.

What are your concerns with it not being a link? Happy to discuss

Reply:
(Also applies to the mobile one below 🙏)

Reply:
Resolved in 657d73f